dark-theme.css

```diff
@@ -4,7 +4,6 @@
   --main-highlight: #545454; /* Message bar top (new messages), Currently selected chat, Nes messages line */
   --main-text: #e6e6e6; /* Main text in the chat windows */
   --secondary-text: #e6e6e6B3; /* Secondary text (70% of main text) */
-  --sidebar-bg-color: #222; /* Sidebar background color */
```

Comment:
Could I ask why you decided to not have the possibility for its own sidebar-bg-color?

That would allow people to define different colors for sidebar than for the main message view. 

I for example use this heavily. Also I would love to add another color for the thread view to have more depth in this case. see the message on:  https://github.com/LanikSJ/slack-dark-mode/issues/103#issuecomment-518358020

I'd not look into it though if that is not what you'd like to move forward with @LanikSJ

Reply:
That's totally my flawed thinking. I figured if the color codes are the same why not combine them, but what I failed to think of was if somebody wanted to change sidebar colors only. I'm going to put that code back actually. Thanks for catching it. :+1:

Reply:
I'm using this at the moment for the sidebar btw:
`  --sidebar-bg-color: rgb(45, 45, 45); /* Sidebar background color */`
![image](https://user-images.githubusercontent.com/479738/62489081-7a32d680-b78b-11e9-92b5-3692b61653c1.png)

Reply:
Oh im sorry, i totally misunderstood the conversation.  Nevermind me!

Reply:
@LanikSJ thank you so much :).